docs/pandas_supported.rst

```diff
@@ -16,9 +16,32 @@ about an API or method in Modin, please contact the Modin `developer mailing lis
 API Completeness
 ----------------
 
+.. raw:: html
+
+   <img src="https://img.shields.io/badge/pandas%20api%20coverage-71.77%25-orange.svg">
+
+Currently, we support ~71% of the pandas API. The exact methods we have implemented are
+listed below.
+
 We have taken a community-driven approach to implementing new methods. We did a `study
 on pandas usage`_ to learn what the most-used APIs are. We currently support **93%** of
-the pandas API based on usage, and are actively expanding the API.
+the pandas API based on our study on pandas usage, and are actively expanding the API.
+
+Defaulting to pandas
+--------------------
+
+The remaining unimplemented methods default to pandas. This is done to allow workloads
+that use some of the methods that are not yet implemented to continue using Modin. Here
```

Comment:
Maybe change the second sentence to "This allows users to continue using Modin even though their workloads contain function not yet implemented in Modin." (or something like that - mainly for readability)